apps/api/views/profiles.py

```diff
@@ -44,14 +45,30 @@ class TestStudentViewSet(ModelViewSet):
     serializer_class = serializers.profiles.TestStudentSerializer
 
 
+def make_ordinal(n):
```

Comment:
Used to render the index of the student with an error to a human-readable ordinal string.